src/schemas/CreditCardResponse.yaml

```diff
@@ -21,8 +22,8 @@ properties:
     example: 2025
   last_digit:
     type: string
-    example: "7890"
+    example: '7890'
```

Comment:
It changes something on typecript side ?

Reply:
nope it's just my prettier, i can revert

Reply:
https://github.com/Qovery/qovery-openapi-spec/pull/499/commits/05dfdd171a14481898218c00b12377f66fb3b5fd